src/pages/about.vue

```diff
@@ -72,7 +72,7 @@
   </VContentPage>
 </template>
 
-<script>
+<script lang="ts">
```

Comment:
Great question, @krysal !

If we include this page in `tsconfig`, then it will be typechecked. And if the types are incorrect, it will not build the app. And because we have some dependencies on Aria-kit that uses some React types internally, we cannot typecheck many of the pages/components that have popups or modals in them.

When we only add `lang="ts"`, but do not add the files to `tsconfig`, we can use Typescript in the code, but type errors will not prevent the app from building. So, we can use the TS type annotations and declare types instead of using the JSDoc comments in these components. I feel it is easier to read the code when all components use the same type annotation style, you don't have to context-switch as much.